test/core/codec/leb128/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright Soramitsu Co., Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+addtest(leb128_test
+        leb128_test.cpp
+    )
+target_link_libraries(leb128_test
```

Comment:
seems like indentation is broken

Reply:
This is negotiated format (approved by Kamil)